src/main/java/nl/beehivejava/scriptj/Plugin.java

```diff
@@ -33,16 +40,22 @@ public Plugin(String location, PluginInformation information) {
     }
 
     /**
-     * Starts this {@code Plugin}.
+     * Starts this {@code Plugin} and all its {@link Script Scripts}.
      *
      * @throws IllegalStateException If this {@code Plugin} is already started, that is if {@link #state()} returns
      *                               {@link PluginState#STARTED}.
+     *                               If this {@code Plugin} does not contain any {@code Scripts}.
      */
     public void start() {
         if (state == PluginState.STARTED) {
             throw new IllegalStateException("This plugin is already started.");
         }
 
+        if (scripts.size() < 1) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1155)